Steps4Impact/LoginV2/LoginV2ViewController.swift

```diff
@@ -277,8 +278,28 @@ extension LoginV2ViewController {
   
   @objc
   private func googleLoginButtonTapped() {
-    GIDSignIn.sharedInstance()?.presentingViewController = self
-    GIDSignIn.sharedInstance()?.signIn()
+    let config = GIDConfiguration(clientID: FirebaseApp.app()?.options.clientID ?? "")
```

Comment:
`clientId ?? ""` -> should this be an error case instead?

Reply:
Yes, I think that should be an error scenario.

Reply:
Updated to be an error